semana-02/todo/add.py

```diff
@@ -0,0 +1,43 @@
+import json
+import uuid
+
+from file_manager import write_file, read_file
+
+def validate_task_struct(task: dict):
+    keys = task.keys()
+    required_keys = ["status", "desc"]
+    diff: set = required_keys - keys
+
+    if len(diff) > 0:
+        print(f"Error: {list(required_keys)} are required in data struct")
```

Comment:
Aqui não é necessário o casting para `list` porque a variável `required_keys` já é uma lista
```suggestion
        print(f"Error: {required_keys} are required in data struct")
```